app/services/group_operation_service.rb

```diff
@@ -0,0 +1,19 @@
+class GroupOperationService
```

Comment:
@hsong-rh 
Do you want to follow the pattern in service portal?
https://github.com/ManageIQ/service_portal-api/blob/master/app/services/service_catalog/service_plans.rb

Reply:
@hsong-rh was following the existing pattern in approval-api. We have pattern like `GroupOperationService#operate` or `WorkflowCreateService#create`. Even though the name appears a little redundant, we feel it is good for readability. We know immediately it is a service class, and method tells what the method intents to do. We don't limit the method name to a fixed one like `process`. It provides us a little more flexibility. For example the same service class may be able to accommodate two slightly different flavors, such as `create` and `create_if`.

We should create another PR to change the coding style for all service methods if we do need to refactor them.